tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs

```diff
@@ -43,21 +47,654 @@ public class SanitizerDictionary
+                    new BodyKeySanitizer("$..to"),
+                    "AZSDK3424"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..from"),
+                    "AZSDK3425"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..outputDataUri"),
+                    "AZSDK3426"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..inputDataUri"),
+                    "AZSDK3427"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..containerUri"),
+                    "AZSDK3428"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..sasUri"),
+                    "AZSDK3429"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..id"),
+                    "AZSDK3430"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..token"),
+                    "AZSDK3431"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..appId"),
+                    "AZSDK3432"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..userId"),
+                    "AZSDK3433"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..id"),
+                    "AZSDK3434"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..storageAccount"),
+                    "AZSDK3435"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..resourceGroup"),
+                    "AZSDK3436"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..guardian"),
+                    "AZSDK3437"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..scan"),
+                    "AZSDK3438"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..catalog"),
+                    "AZSDK3439"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..lastModifiedBy"),
+                    "AZSDK3440"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..managedResourceGroupName"),
+                    "AZSDK3441"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..createdBy"),
+                    "AZSDK3442"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..tenantId"),
```

Comment:
Can this be sanitized with Guid.Empty? The C# mgmt model deserializes this into a Guid - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/src/Resources/Generated/SubscriptionData.Serialization.cs#L164